packages/traceloop-sdk/traceloop/sdk/dataset/dataset.py

```
@@ -0,0 +1,159 @@
+import csv
```

Ruff (0.12.2)

`csv` imported but unused

Remove unused import: `csv`

(F401)